helm/chart/doc.yaml

```diff
@@ -206,7 +206,7 @@
 
 
 ## @param   metricsOperator.manager.image.repository specify registry for manager image
-## @param   metricsOperator.manager.image.tag  select tag for manager image <!---x-release-please-version-->
+## @param   metricsOperator.manager.image.tag select tag for manager image
```

Comment:
so we do not use release please anymore for the versions in the docs?

Reply:
We should not update this automatically, as if we release lifecycle-operator, it does not mean, we will bump the metrics-operator (until now it is like that). We have the same approach for cert-manager here